velox/core/PlanNode.cpp

```diff
@@ -250,6 +429,28 @@ void ProjectNode::addDetails(std::stringstream& stream) const {
   }
 }
 
+folly::dynamic ProjectNode::serialize() const {
+  auto obj = PlanNode::serialize();
+  obj["names"] = ISerializable::serialize(names_);
+  obj["projections"] = ISerializable::serialize(projections_);
+  return obj;
+}
+
+// static
+PlanNodePtr ProjectNode::create(const folly::dynamic& obj, void* context) {
+  auto source = deserializeSingleSource(obj, context);
+
+  auto names = ISerializable::deserialize<std::vector<std::string>>(
```

Comment:
maybe we could also have a tiny `deserializeStrings()` helper to make it a little less verbose?

Reply:
Good suggestion. Applied.